src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,21 @@
+:root {
```

Comment:
All these variables could be static. I mean they will not change. So you could use SCSS variables everywhere (easier to use) instead using CSS custom properties